### PR TITLE
Added "about me" section and buttons to home page

### PR DESCRIPTION
**Pull request by shivanighanta**
Added basic functionality to home page:
- About me section
- Clickable buttons that direct user to home, email and resume (resume to be uploaded later)

screencapture-8080-dot-12463562-dot-devshell-appspot-2020-05-29-15_42_37

### CODE REVIEW

**Comment by jbir789** [comment]
This looks good. However, it's a bit difficult to see what the page looks like from raw HTML / CSS. Could you share a link to the live page so we can see the rendered page? You can follow [these instructions](https://github.com/shivanighanta/my-portfolio/blob/master/walkthroughs/week-2-web-development/portfolio-walkthrough.md#live-server), or alternatively attach a screenshot.

A few more comments:
  - Your PR description is excellent: it is detailed without being too extensive (which is good) and we can immediately know what was changed from reading it.
  - The title could be a bit more descriptive. In the current state, reading the PR title isn't enough to understand what the PR addresses (which functionality is being added?). This is important, because people search for PRs / commits by looking at the first line / title, and not the description. My recommendation is to have a title that's a good descriptive 1-line description of the PR, and take advantage of the description to fill in any extra info relevant to the change.

**Comment by shivanighanta** [comment]
Ok thanks! I added a more detailed title and a screenshot in the original comment.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,130 @@
...
+</div>
+      </nav>
+
+      <!-- Title Content & Bio -->
+      <div class="bitmoji row">
+              <div class="container">
+                  <div class="row">
+                      <div class="about">
+                          <div id="photos">
+                              <center><img id = "bitmoji" src="bitmoji-icon.png" class="img-responsive" height="420" width="420"/></center>
```
Comment (ginellegaisano) [inline portfolio/src/main/webapp/index.html:51]: I noticed in the screenshot that the images do not show - could you also include the image files in this PR and update the screenshot?

**Comment by jbir789** [comment]
Is it intended that the pictures aren't displayed? I see the PR includes one image, so just making sure it wasn't supposed to be shown as part of the change.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,130 @@
...
+            <li><a href="mailto:shivani.ghanta@gmail.com">Email Me</a></li>
+            <li><a href="resume.pdf">Resume</a></li>
+        </ul>
     </div>
+
+</div>
+      </nav>
+
+      <!-- Title Content & Bio -->
+      <div class="bitmoji row">
```
Comment (ginellegaisano) [inline portfolio/src/main/webapp/index.html:46]: To confirm: I'm not sure what bitmoji refers to - should this be a info-row like the following sections?

Comment (shivanighanta) [reply portfolio/src/main/webapp/index.html:46]: For the bitmoji row I wanted to do formatting that is different from the info row in the CSS file so I made them separate.

**Comment by shivanighanta** [comment]
Yes, the page is currently not supposed to show any images, I was going to add them later.

**Comment by jbir789** [comment]
Thanks for the explanation! While it's great to have incremental changes / PRs, it's typically preferred for each change to be functional to some extent. For example, if you have a PR with images that cannot be resolved, it feels like there is a bug in the implementation to the reviewer.

What I would suggest is removing the images tags from the page for now, and add them back when the images are added. Alternatively, you can add the missing images to this PR. What do you think?

**Comment by shivanighanta** [comment]
Got it! I added the images. The updated screenshot is below
screencapture-8080-dot-12463562-dot-devshell-appspot-2020-05-29-15_42_37

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -6,4 +6,4 @@
 
 #greeting-container {
   margin-top: 20px;
-}
+}
```
[the remark this answers is not in the document]
Comment (shivanighanta) [reply portfolio/src/main/webapp/style.css]: Fixed!

**Pull request merged**